Gems/Atom/Feature/Common/Code/Source/AuxGeom/FixedShapeProcessor.cpp

```diff
@@ -1303,7 +1303,7 @@ namespace AZ
             objectBuffers.m_pointIndexCount = static_cast<uint32_t>(meshData.m_pointIndices.size());
             AZ::RHI::SingleDeviceIndexBufferView pointIndexBufferView =
             {
-                *objectBuffers.m_pointIndexBuffer,
+                *objectBuffers.m_pointIndexBuffer->GetDeviceBuffer(RHI::MultiDevice::DefaultDeviceIndex),
```

Comment:
Maybe DefaultDeviceIndex can be the default param so you dont have to pass it in every time.

Reply:
Please read the second bullet point in the description of the PR. This is one of those cases. We actually want to keep the `DefaultDeviceIndex` because it helps us find those places where we are not done yet.